cdap-security/src/main/java/co/cask/cdap/security/store/DefaultSecureStoreService.java

```diff
@@ -77,11 +72,11 @@
   @Override
   public final Map<String, String> listSecureData(String namespace) throws Exception {
```

Comment:
The javadoc has to be updated to change `Returns an empty list if the user does not have access to the namespace or any of the keys in the namespace.` => `Returns an empty list if the user does not have access to any of the keys in the namespace.`